docs/design/arc-onboarding.md

```diff
@@ -0,0 +1,27 @@
+# Azure Arc for Kubernetes integration
+
+## Motivation
+
+Simplify Arc agents deploymet by making it an optional last step of the cluster deployment process.
```

Comment:
s/deploymet/deployment

Reply:
Also, should be `agents' deployment` if we're talking about multiple agents, or `agent's deployment` if we're talking about a single agent